app/uk/gov/hmrc/softdrinksindustrylevy/controllers/VariationsController.scala

```diff
@@ -74,4 +93,42 @@ class VariationsController @Inject() (
 
       }
     }
+
```

Comment:
do we need all these details for this event? Hope address would be ok to log here? but I don't think it's required in this case?